examples/nlp/duplex_text_normalization/duplex_text_normalization_test.py

```diff
@@ -50,11 +52,11 @@
 
 
 from helpers import DECODER_MODEL, TAGGER_MODEL, instantiate_model_and_trainer
-from nltk import word_tokenize
 from omegaconf import DictConfig, OmegaConf
 
 import nemo.collections.nlp.data.text_normalization.constants as constants
 from nemo.collections.nlp.data.text_normalization import TextNormalizationTestDataset
+from nemo.collections.nlp.data.text_normalization.utils import basic_tokenize
```

Comment:
whats difference between basic_tokenize and word_tokenize?

Reply:
`basic_tokenize` takes as inputs the input text and the language. If the language is English, then `basic_tokenize` will just call `word_tokenize`. For other languages (e.g., ru and de),  `basic_tokenize` will just split words between spaces.